src/site/stages/build/drupal/page.js

```diff
@@ -388,6 +412,13 @@ function compilePage(page, contentData) {
       break;
   }
 
+  if (entityUrl.path === '/resources') {
+    pageCompiled = mergeTaxonomiesIntoResourcesAndSupportHomepage(
+      pageCompiled,
+      allTaxonomies,
+    );
+  }
```

Comment:
Here is where the taxonomies data is merged into the R&S homepage.

Reply:
I wanted to add this logic into the build step at `src/site/stages/build/plugins/create-resources-and-support-section.js` but the build steps after this one don't have access to the original Drupal data structure, so I can't access the `allTaxonomies` from there...so the logic had to go in this lengthy file.

Reply:
Interesting, I was wondering how this would be achieved! Super cool! 💯